CleverTapSDK/Inbox/controllers/CleverTapInboxViewController.m

```diff
@@ -162,6 +162,17 @@ - (void)setUpInboxLayout {
     
     if (_config && _config.navigationTintColor) {
         self.navigationController.navigationBar.titleTextAttributes = @{NSForegroundColorAttributeName : _config.navigationTintColor};
+    } else {
+        self.navigationController.navigationBar.titleTextAttributes = @{NSForegroundColorAttributeName : [UIColor blackColor]};
+    }
+    
+    //Added fix for iOS 15 beta to update navigationBarTintColor and navigationTintColor
+    if (@available(iOS 13.0, *)) {
+        UINavigationBarAppearance *appearance = [UINavigationBarAppearance new];
+        appearance.backgroundColor = (_config && _config.navigationBarTintColor) ? _config.navigationBarTintColor : [UIColor whiteColor];
+        appearance.titleTextAttributes = @{NSForegroundColorAttributeName : (_config && _config.navigationTintColor) ? _config.navigationTintColor : [UIColor blackColor]};
+        self.navigationController.navigationBar.standardAppearance = appearance;
+        self.navigationController.navigationBar.scrollEdgeAppearance = self.navigationController.navigationBar.standardAppearance;
     }
```

Comment:
Could it be possible to write generic code for navigation bar UI handling? A way to mitigate iOS versions if condition(s)